### DISCUSSION TITLE
Go check my LMMS tutorials!

### DISCUSSION THREAD

**Discussion by ghost** [LMMS/lmms#6737] [General]
HOW TO MAKE "STAY" BY THE KID LAROI & JUSTIN BIEBER : https://www.youtube.com/watch?v=HmgPirgJqjU&ab_channel=Mr.LMMS

Don't hesitate to subscribe, to like and to comment my videos.
Don't hesitate if you encounter a problem :)
ENJOY!

**Comment by irrenhaus3** [comment]
Hi, welcome to the github repo. If you want to promote your stuff, please do it in the appropriate location. There is a subreddit, a discord server and a forum, all reachable via https://lmms.io/get-involved.  
Github is the platform where development of the software itself is coordinated, not a place for posting or promoting content. 
Thanks!

**Comment by zonkmachine** [comment]
Closing, please use the channels suggested above!

Thanks!
